### PR TITLE
✨ Update Waiting Room

### PR DESCRIPTION
**Pull request by SteffenKn**
## What did you change?

This PR adjusts the waiting room to the management api.
Also the Notifications for not supported form field types has been improved.

## How can others test the changes?

- Open up the BPMN-Studio
- Run any diagram with user tasks
- **Enjoy the waiting room**
- Wait for the user task
- Confirm the user task
- **Enjoy the waiting room**
- Notice that one gets redirected after the process has been finished

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/modules/process-list/process-list.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ export class ProcessList {
     this._getProcessesIntervalId = window.setInterval(async() => {
       await this.updateProcesses();
       this.updateList();
-    }, environment.processengine.poolingInterval);
+    }, environment.processengine.pollingInterval);
```
Comment (cmg-dev) [inline src/modules/process-list/process-list.ts]: please create an issue, that we NEED a ConfigurationService.

This direct access to a config is very fragile and can lead to strange user experience-- that is sadly hard to debug.

File: src/modules/waiting-room/waiting-room.ts [hunk 2]
```diff
@@ -1,51 +1,121 @@
...
 export class WaitingRoom {
 
   private _router: Router;
-  private _managementApiClient: ManagementApiClientService;
-  private _processModelId: string;
+  private _correlationId: string;
   private _notificationService: NotificationService;
+  private _authenticationService: IAuthenticationService;
+  private _managementApiClient: IManagementApiService;
+  private _pollingTimeout: NodeJS.Timer;
```
Comment (cmg-dev) [inline src/modules/waiting-room/waiting-room.ts]: isn't this a `_pollingTimer`?

File: aurelia_project/environments/stage.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ export default {
     authority: 'http://localhost:5000',
   },
   processengine: {
-    poolingInterval: 10000,
+    pollingInterval: 10000,
```
Comment (cmg-dev) [inline aurelia_project/environments/stage.ts]: @Paulomart fix das bitte

Comment (cmg-dev) [reply aurelia_project/environments/stage.ts]: add a unit.

Comment (Paulomart) [reply aurelia_project/environments/stage.ts]: I will address this in a future pr.

**Pull request merged**